src/Main.java

```diff
@@ -1,5 +1,73 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        //Задача 1
+
+        byte age = 15;
+
+        if (age >=18) {System.out.println("Если возраст человека равен "+age+", то он совершеннолетний");}
+        else {System.out.println("Если возраст человека равен "+age+", то он не достиг совершеннолетия, нужно немного подождать");}
+
+        //Задача 2
+
+        byte temp=5;
+        boolean weatherCondition= temp>=5;
+
+        if (weatherCondition) {System.out.println("Сегодня тепло, можно идти без шапки");}
+        else {System.out.println("На улице холодно, нужно надеть шапку");}
+
+        //Задача 3
+
+        short speed=59;
+
+        if (speed>=60) {System.out.println("Если скорость "+speed+" то придется заплатить штраф");}
+        else {System.out.println("Если скорость "+speed+" то можно ездить спокойно");}
+
+        //Задача 4
+
+        byte age4=60;
+
+        if ( age4<2) {System.out.println("Если возраст человека равен " +age4+", то ему пора спать");}
+        else if (age4>=2 && age4<7) {System.out.println("Если возраст человека равен " +age4+", то ему нужно ходить в детский сад");}
+        else if (age4>=7 && age4<18) {System.out.println("Если возраст человека равен " +age4+",  то ему нужно ходить в школу.");}
+        else if (age4>=18 && age4<24) {System.out.println("Если возраст человека равен " +age4+",  то его место в университете");}
+        else if (age4>=24 && age4<60) {System.out.println("Если возраст человека равен " +age4+", то ему пора ходить на работу");}
+        else {System.out.println("Если возраст человека равен " +age4+", то он может отдохнуть");}
+
+        //Задача 5
+
+        byte age5=14;
```

Comment:
зачем заводить новую переменную, если можно просто переиспользовать уже объявленную?
```suggestion
        age = 14;
```